core/templates/app.constants.ts

```diff
@@ -107,7 +107,7 @@ export const AppConstants = {
 
   // Flag to toggle the support for a new state to direct the learners
   // to if they get stuck.
-  DEST_IF_REALLY_STUCK_FEAT_ENABLED: false,
+  DEST_IF_REALLY_STUCK_FEAT_ENABLED: true,
```

Comment:
Since we're enabling this feature completely, we can just remove this flag and the condition.

Reply:
Fixed.